server/src/main/java/org/cloudfoundry/identity/uaa/scim/vmidentity/VmidentityScimUserProvisioning.java

```diff
@@ -104,115 +111,59 @@ public ScimUser delete(String id, int version) {
     }
 
     @Override
-    public List<ScimUser> query(String filter)
-    {
-        logger.debug(String.format("query called with filter '%s'", filter));
+    public List<ScimUser> query(String filter) {
```

Comment:
q: will the basic filter (like originally supported userid=x) be on part perf wise still ? like original retrievebyid ? this is important t ensure since evidently pcf was doing these user lookups, via this filter method .... 
?